cookbook/bundles/best_practices.rst

```diff
@@ -71,49 +68,51 @@ class name.
     :class:`Symfony\\Bundle\\FrameworkBundle\\FrameworkBundle`.
 
 Each bundle has an alias, which is the lower-cased short version of the bundle
-name using underscores (``acme_hello`` for ``AcmeHelloBundle``, or
-``acme_social_blog`` for ``Acme\Social\BlogBundle`` for instance). This alias
-is used to enforce uniqueness within a bundle (see below for some usage
-examples).
+name using underscores (``acme_blog`` for ``AcmeBlogBundle``). This alias
+is used to enforce uniqueness within a project and for defining bundle's
+configuration options (see below for some usage examples).
 
 Directory Structure
 -------------------
 
-The basic directory structure of a HelloBundle must read as follows:
+The basic directory structure of an AcmeBlogBundle must read as follows (if you
+use `PSR-4`_ the ``Acme/`` and ``BlogBundle/`` folders might not exist):
 
 .. code-block:: text
 
-    XXX/...
-        HelloBundle/
-            HelloBundle.php
-            Controller/
-            Resources/
-                meta/
-                    LICENSE
-                config/
-                doc/
-                    index.rst
-                translations/
-                views/
-                public/
-            Tests/
-
-The ``XXX`` directory(ies) reflects the namespace structure of the bundle.
-
-The following files are mandatory:
-
-* ``HelloBundle.php``;
-* ``Resources/meta/LICENSE``: The full license for the code;
+    <your-project>/
+    ├─ ...
+    └─ src/
+       └─ Acme/
+          └─ BlogBundle/
+              ├─ AcmeBlogBundle.php
+              ├─ Controller/
+              ├─ README.md
+              ├─ Resources/
+              │   ├─ meta/
+              │   │  └─ LICENSE
+              │   ├─ config/
+              │   ├─ doc/
+              │   │  └─ index.rst
+              │   ├─ translations/
+              │   ├─ views/
+              │   └─ public/
+              └─ Tests/
+
+**The following files are mandatory**, because they ensure a structure convention
+that automated tools can rely on:
+
+* ``AcmeBlogBundle.php``: This is the class that transforms a plain directory
+  into a Symfony bundle;
+* ``README.md``: This file contains the basic description of the bundle and it
```

Comment:
we should allow to use other formats supported by Github for the readme (or whatever project hosting you use).
Otherwise some SensioLabs bundles would have to be considered invalid when they use ``README.rst``, while there is no reason for it. The requirement is to provide a readme for the project, but writing it in Markdown is not mandatory